packages/governance/src/contractHelper.js

```diff
@@ -46,6 +46,12 @@ const handleParamGovernance = (zcf, paramManager) => {
     getUnknown: paramManager.getUnknown,
   };
 
+  /**
+   * @param {T} originalPublicFacet
+   * @returns {T & GovernedPublicFacet}
+   * @template T
+   */
+  // @ts-expect-error bounded polymorphism doesn't fit well in JSDoc
   const wrapPublicFacet = (originalPublicFacet = {}) => {
```

Comment:
I actually used an inline cast for fixing these, but I know we frown upon them

```suggestion
  const wrapPublicFacet = (originalPublicFacet = /** @type {T} */ ({})) => {
```

Reply:
Thanks for the suggestion. In looking at the context, @michaelfig and I discovered a good type that's a bit more involved.